backend/src/passwords.cpp

```diff
@@ -0,0 +1,72 @@
+#include <iostream>
+#include "passwords.hpp"
+
+namespace passwords
+{
+    std::unordered_set<std::string> generatePasswords(const int &numPasswords, const int &maxChars)
```

Comment:
**issue**: You don't need to use const reference with primitive data types. Also, parameter names are snake_case, not camelCase.